lib/unsplash/photo.rb

```diff
@@ -52,17 +52,19 @@ def find(id)
       # @param user [String] Limit selection to given User's ID.
       # @param query [String] Limit selection to given search query.
       # @param orientation [String] Filter by orientation of the photo. Valid values are landscape, portrait, and squarish.
+      # @param content_filter [String] Filter by content_filter to remove content that may be unsuitable for younger audiences.
```

Comment:
```suggestion
      # @param content_filter [String] Limit results by content_filter to avoid content that may be unsuitable for younger audiences. Valid values are low, high. Defaults to low.
```

Reply:
@internethostage - Changes done as per review.